.github/workflows/e2e.yml

```diff
@@ -0,0 +1,56 @@
+name: E2E CI
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
```

Comment:
Can we use job name other than `build`?
What's the expected duration for this run? Default timeout is 6 hours, and I think it's long to see something is wrong.
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes

Can you set shorter timeout?